### PR TITLE
Add user count REST route

### PR DESCRIPTION
**Pull request by feeds**
First part of #193
Duplicate of #214 (was branched from old staging)

@andyz02 you can call /accounts/count to get a JSON response with a dict with the key 'user_count'. Branch (git checkout number-supporters und git checkout -b NAME ) from number-supporters to implement the UI part.

DON'T MERGE before checking load on server (@feeds Can we create a high number of users now?)

### CODE REVIEW

**Comment by feeds** [comment]
@Baschdl I created a test case for the url and also added a 'many-users-fixture' with about 7000 users.
You can user this to test the load on the server

File: backend/apps/accounts/generate_users.py [hunk 1]
```diff
@@ -61,12 +61,16 @@
...
+from django.contrib.admin.views.decorators import staff_member_required
 
 
 def delete_fakes():
     User.objects.filter(email__contains='email').delete()
 
-
+@login_required
+@staff_member_required
 def populate_db(request):
```
Comment (josauder) [inline backend/apps/accounts/generate_users.py:69]: We should not push this as it is. Both here and in the `urls.py` route, we should use something like:

```
if settings.DEBUG:
       def populate_db(...)
```

**Review by josauder — DISMISSED** [review]

**Comment by feeds** [comment]
@josauder I included you change requests

File: backend/apps/accounts/views.py [hunk 2]
```diff
@@ -245,6 +248,18 @@ def validate_email(request):
         request.user.save()
     return HttpResponseRedirect("/mapview")
 
+class UserCountView(APIView):
```
Comment (kevihiiin) [inline backend/apps/accounts/views.py:251]: Maybe add restriction for the usercount aswell?
Not sure how usefull the normal staff_required one is for REST calls, but maybe [tokens](https://simpleisbetterthancomplex.com/tutorial/2018/11/22/how-to-implement-token-authentication-using-django-rest-framework.html)?

**Review by kevihiiin — CHANGES_REQUESTED** [review]
Looks good and works on my machine

**Review by kevihiiin — APPROVED** [review]

**Pull request merged**